.gitignore

```diff
@@ -272,3 +272,6 @@ bh_unicode_properties.cache
 # Sublime-github package stores a github token in this file
 # https://packagecontrol.io/packages/sublime-github
 GitHub.sublime-settings
+
+# Julia Manifest file
+Manifest.toml
```

Comment:
Nice catch.
 
Note, this file is generated via the template file in

https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/.gitignore


This file isn't affected by `npm run build` (unlike `src/DashTextareaAutocomplete.jl` below), so this modif isn't at risk of getting overridden, but still it would be nice to submit a patch over on https://github.com/plotly/dash-component-boilerplate to help out the open-source community :smile: